src/robusta/core/sinks/robusta/dal/supabase_dal.py

```diff
@@ -96,27 +104,79 @@ def __init__(
         self.sign_in()
         self.sink_name = sink_name
         self.signing_key = signing_key
+        self.event_loop = event_loop
+
+    def __to_db_scanResult(self, scanResult: ScanReportRow) -> Dict[Any, Any]:
+        db_sr = scanResult.dict()
+        db_sr["account_id"] = self.account_id
+        db_sr["cluster_id"] = self.cluster
+        return db_sr
+    
+
+    def persist_scan(self, enrichment: Enrichment):
+
+        for block in enrichment.blocks:
```

Comment:
Would it be better to persist all scan report blocks?
like:
```
for block in emrichments.blocks if isinstance(block, ScanReportBlock):
  # persist scan
```
We're not supporting the case there are multiple blocks in the enrichment
Is it too restrictive ?

Reply:
yea , was a mistake